webpack/scenes/ContentViews/Table/ContentViewsTable.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+} from '@patternfly/react-table';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+import Loading from './Loading';
+import EmptyStateMessage from '../components/EmptyStateMessage';
+import tableDataGenerator from './tableDataGenerator';
+import './ContentViewsTable.scss';
+
+const ContentViewTable = ({
+  loadContentViewDetails, detailsMap, results, loading,
+}) => {
+  const [table, setTable] = useState({ rows: [], columns: [] });
+  // Map of CV id to expanded cell, if id not present, row is not expanded
+  const [expandedColumnMap, setExpandedColumnMap] = useState({});
+  const cvsPresent = results && results.length > 0;
```

Comment:
I wonder if we can move this to a selector? so the table will already receive `cvsPresent` as a prop